modules/35_transport/edge_esm/sets.gms

```diff
@@ -122,7 +122,7 @@ ConvCase
 ConvCaseWise
 ElecEra
 ElecEraWise
-HydrHype
+HydrHype4
```

Comment:
does this mean that it only works with new input data? Does it give an error for input data containing "HydrHype" ?

Reply:
`HydrHype` is deprecated and not produced by `mrremind` since quite a while. Can be a problem if one tries to use new REMIND with old input data. Does this happen?

Reply:
And there are still config files that use it, see: `grep ";HydrHype;" config/*.csv config/*/*.csv`.